cmd/tester/block_generator.go

```diff
@@ -380,6 +382,13 @@ func NewBlockGenerator(ctx context.Context, outputFile string, initialHeight int
 	}
 	bg.forkId = forkid.NewID(blockchain)
 
+	if err := output.Flush(); err != nil {
```

Comment:
Could you encapsulate it in something like

```
func Reopen(f *os.File, name string) (*os.File, error) {
	if err := f.Flush(); err != nil {
		return nil, err
	}
	if err := f.Close(); err != nil {
		return nil, err
	}

	return os.Open(name)
}

Reply:
Actually I think opposite: need split func to 2 - then each one will look like normal go pattern: 
f.Open()
defer f.close()

or
a := newBufIo() 
defer a.Flush()